pom.xml

```diff
@@ -223,6 +223,15 @@
             </lifecycleMappingMetadata>
           </configuration>
         </plugin>
+
+        <!-- Inject the "incubating" into the output filename -->
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-assembly-plugin</artifactId>
+          <configuration>
+            <finalName>edgent-incubating-${project.version}</finalName>
```

Comment:
It was intentional ... The POM is named edgent-parent, but the source release contains the entire project. That's why I omitted the "parent" from the name.